project/src/const.js

```diff
@@ -3,8 +3,7 @@ export const AppRoute = {
   SIGN_IN: '/login',
   MY_LIST: '/mylist',
   FILM: '/films/:id',
-  FILM_DETAILS: '/films/:id/:tab',
-  FILM_REVIEWS: '/films/:id/:tab',
+  FILM_DETAILS_OR_REVIEWS: '/films/:id/:tab(details|reviews)',
```

Comment:
если бы ты поставил вопросительный знак, то массива в руте не нужно

```suggestion
  FILM_DETAILS_OR_REVIEWS: '/films/:id/:tab(details|reviews)?',
```